.github/workflows/ci.yml

```diff
@@ -1,18 +1,15 @@
-name: PR CI
+name: Name Request UI CI
 
 on:
   pull_request:
     types: [assigned, synchronize]
 
-env:
-  IMAGE_NAME: namerequest-ui
-
 jobs:
+  name-request-ui-ci:
+    runs-on: ubuntu-latest
 
-  build:
     env:
       working-directory: ./client
```

Comment:
Are these 2 lines needed? (They are not in the CI scripts for some of the other projects.)